EventPlanning.DAL/Repository.cs

```diff
@@ -53,6 +54,21 @@ public bool SaveActivityType(ActivityType activityType)
             return true;
         }
 
+        public IQueryable<Event> GetEvents(string pattern)
```

Comment:
Is there any difference between `IQueryable` and `IEnumerable` in this case? https://stackoverflow.com/questions/2876616/returning-ienumerablet-vs-iqueryablet